bucket/ezunlock.json

```
@@ -0,0 +1,33 @@
+{
+    "version": "v0.1",
+    "description": "A simple GUI application for deleting stuck/stale/undeletable files.",
+    "homepage": "https://github.com/demberto/EzUnlock",
+    "license": "MIT",
+    "architecture": {
+        "64bit":{
+            "url": "https://github.com/demberto/EzUnlock/releases/download/v0.1/EzUnlock_x64_v0.1.zip",
+            "hash": "0f4063e5b4ffff6aaf6806386d591cf621ae76a6637f91bdccbda2e1021a68c3",
+        },
+        "32bit":{
+            "url": "https://github.com/demberto/EzUnlock/releases/download/v0.1/EzUnlock_x86_v0.1.zip",
+            "hash": "e4e6006634183af3dc5b4825e01a71909c5eda2eef15f62ee094e5baf554c55b",
+        }
+    },
+    "shortcuts": [
+        [
+            "EzUnlock.exe",
+            "EzUnlock"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit":{
+                "url": "https://github.com/demberto/EzUnlock/releases/download/v$verison/EzUnlock_x64_v$version.zip",
+            },
+            "32bit":{
+                "url": "https://github.com/demberto/EzUnlock/releases/download/v$verison/EzUnlock_x64_v$version.zip",
+            }
+        }
+    }
```

```suggestion
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/demberto/EzUnlock/releases/download/v$version/EzUnlock_x64_v$version.zip"
            },
            "32bit": {
                "url": "https://github.com/demberto/EzUnlock/releases/download/v$version/EzUnlock_x64_v$version.zip"
            }
        }
    }
```

* There is a needless comma at the end of the line, which will cause **the JSON file fail to load**.
* typo: `$verison` -> `$version`